### ISSUE TITLE
Failure to update the Members Page

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1372]
It used to be we would inform Members when the software changed and why when the software was updated, This information has not been updated since August 2017.
We should either update it or decide we will not do it and remove the current information

**Comment by Captainkirkdawson** [comment]
![updating.JPG](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/53ecbbfc82b317f7d9ad93dc/125516c66e82c728ace21e0d46fd849e22968aa788abf301b44fff9e2c4533317346f6787586e22de61b2f595f581ca9415a5f44fdfadc2ba5e02e6c981e5ab38532022d8c6853beb2901eb80bbb258a4b82b0b9884fc1ed3b2cf5c6.JPG)

**Comment by SteveBiggs** [comment]
I agree, but I believe that Rich is looking at introducing a new software versioning system at some point and perhaps waiting for this is why no release notes have been added since August .....?

**Comment by Captainkirkdawson** [comment]
The github versioning process that was the basis of that idea is problematic as FR and FC share the same basic code base

**Comment by Captainkirkdawson** [comment]
Now being done

**Comment by PatReynolds** [comment]
I can't remember seeing this page on refinery before.  Maybe I have a different view now?

**Comment by Captainkirkdawson** [comment]
The page has been there ever since day 1. It is the page that is displayed as part of the landing page on login. Initially I updated the page after each sprint deployment. When responsibility for deployment moved to the staff some 18 months ago (when I went on one of my extended vacations). The updating of the page ceased. It restarted when the release notes process was adopted as part of the sprint deployment. It is only updated on production never on test3
